Gopkg.toml

```diff
@@ -73,7 +73,8 @@
 
 [[constraint]]
   name = "github.com/rjeczalik/notify"
-  version = "0.9.2"
+  revision = "9ace999fc73bca667fd45dafc41356186f8b7e65"
+  source = "https://github.com/dgageot/notify.git"
```

Comment:
can you add a comment here explaining what you fixed and why we're using your fork?

also, did you send a PR? :)

Reply:
I’ll do that! Thanks!

Reply:
In fact, with the mitigation code, I might get rid of the fork!

Reply:
I got rid of the fork